### ISSUE TITLE
Condense excess commands into the dialog node

### ISSUE THREAD

**Issue by sebulba69** [AnidemDex/Godot-DialogPlugin#96]
### Request

The comment, set variable, goto, and return commands all do not need to be their own separate commands and can be condensed into the dialog command.

The fact they're all separate means there's an insane amount of clutter on the timeline when there really doesn't need to be. The only thing I want to see on my timeline is the dialog I've written.

### Reference

Here's a crude MS Paint mockup of an idea for how to condense these nodes into one.
![image](https://github.com/AnidemDex/Godot-DialogPlugin/assets/72589301/040bab2b-74ce-4e66-b33f-249f6283cc94)

### Additional information

You can also make an argument to move the wait time into the nodes with a default time of 0 or something to indicate you don't need to wait by default before moving. Although, it's still nice to have this as a separate node, so if this is done, please let me use Wait as a separate option.

**Comment by AnidemDex** [comment]
Interesting idea. Commands are now being worked on https://github.com/AnidemDex/Blockflow but this is strictly related to dialog commands.

What do you think @Crystalwarrior ? Blockflow has CommandCollections and we can group commands in a collection to show it as a single command

**Comment by Crystalwarrior** [comment]
> Interesting idea. Commands are now being worked on https://github.com/AnidemDex/Blockflow but this is strictly related to dialog commands.
> 
> What do you think @Crystalwarrior ? Blockflow has CommandCollections and we can group commands in a collection to show it as a single command

Yee, I also remember we discussed merging the "Wait" command into all commands innately so any command can cause a delay before letting the next thing process.
